### ISSUE TITLE
Nuevo estudio: Authentic Illusions

### ISSUE THREAD

**Issue by afk-mario** [indi-es/estudios#128]
## Gracias a: afk - afk@ellugar.co

```json
{
  "name": "Authentic Illusions",
  "country": "México",
  "city": "Monterrey",
  "state": "Nuevo León",
  "website": "http://www.jara-ai.com/",
  "twitter": "https://twitter.com/JARAaiSolutions",
  "facebook": "https://www.facebook.com/JARA.AI.Solutions",
  "instagram": "",
  "tags": [],
  "inactive": false,
  "last_time_active": "2021-11-02T21:31:32.901Z"
}
```

**Comment by Ucamo** [comment]
Part of #133

**Comment by afk-mario** [comment]
closed #133
